dashboard_register.go

```diff
@@ -37,6 +37,14 @@ type HTTPDashboardHandler struct {
 }
 
 func reLogin() {
+	if !config.Global.UseDBAppConfigs {
+		log.WithFields(logrus.Fields{
+			"prefix": "main",
+		}).Info("Skipping reLogin as UseDBAppConfigs is set to false.")
```

Comment:
I would honestly not log this, as this should be transparent to the user (and the message itself is internal).